extlinks/aggregates/management/helpers/aggregate_archive_command.py

```diff
@@ -0,0 +1,307 @@
+        )
+        dump_parser.add_argument(
+            "-f",
+            "--from",
+            nargs="?",
+            type=lambda arg: datetime.datetime.strptime(arg, "%Y-%m").date(),
+            help="A date formatted as YYYY-MM to begin archiving from.",
+            required=True,
+        )
+        dump_parser.add_argument(
+            "-t",
+            "--to",
+            nargs="?",
+            type=lambda arg: datetime.datetime.strptime(arg, "%Y-%m").date(),
+            help="An optional date formatted as YYYY-MM to end archiving from.",
+            required=False,
+        )
+        dump_parser.add_argument(
+            "-o",
+            "--output",
+            nargs="?",
+            type=str,
+            help=f"The directory that the archives containing the {self.name} data should be written to.",
+        )
+        dump_parser.add_argument(
+            "-c",
+            "--container",
+            nargs="?",
+            type=str,
+            help=f"The Swift container to upload {self.name} archives to. If left unspecified then nothing will be uploaded.",
+        )
+
+        load_parser = subparsers.add_parser(
+            "load",
+            help=f"Import {self.name} data into the database from gzipped JSON files.",
+        )
+        load_parser.add_argument(
+            "filenames",
+            nargs="*",
+            type=str,
+            help=f"{self.name} archive filenames to load.",
+        )
+
+        upload_parser = subparsers.add_parser(
+            "upload",
+            help=f"Upload {self.name} data to Swift object storage from the local filesystem.",
+        )
+        upload_parser.add_argument(
+            "-c",
+            "--container",
+            nargs="?",
+            type=str,
+            help=f"The Swift container to upload {self.name} archives to.",
+            required=True,
+        )
+        upload_parser.add_argument(
+            "filenames",
+            nargs="*",
+            type=str,
+            help=f"{self.name} archive filenames to upload.",
+        )
+
+    def handle(self, *args, **options):
+        subcommand = options["subcommand"]
+
+        if subcommand == "dump":
+            self.dump(
+                start=options["from"],
+                end=options["to"],
+                output=options["output"],
+                container=options["container"],
+            )
+        elif subcommand == "load":
+            self.load(filenames=options["filenames"])
+        elif subcommand == "upload":
+            self.upload(container=options["container"], filenames=options["filenames"])
+
+    def dump(
+        self,
+        start: datetime.date,
+        end: Optional[datetime.date] = None,
+        output: Optional[str] = None,
+        container: Optional[str] = None,
+    ):
+        """
+        Dump aggregate data to gzipped JSON files that are grouped by month,
+        and then delete them from the database.
```

Comment:
Nit: Let's add documentation for the parameters

Reply:
Done